Formula/mobile-shell.rb

```diff
@@ -3,7 +3,7 @@ class MobileShell < Formula
   homepage "https://mosh.org"
   url "https://mosh.org/mosh-1.2.6.tar.gz"
   sha256 "7e82b7fbfcc698c70f5843bb960dadb8e7bd7ac1d4d2151c9d979372ea850e85"
-  revision 3
+  revision 4
```

Comment:
Probably no need for a `revision` here. The installed binary will be the same, and that's the criteria for a `revision`.